plugins/com.google.cloud.tools.eclipse.util/src/com/google/cloud/tools/eclipse/util/ArtifactRetriever.java

```diff
@@ -136,15 +136,16 @@ private ArtifactRetriever(String repositoryUrl) {
   /**
    * Returns the latest published release artifact version, or null if there is no such version.
    */
-  public ArtifactVersion getLatestArtifactVersion(String groupId, String artifactId) {
-    return getLatestReleaseVersion(idToKey(groupId, artifactId), null);
+  public ArtifactVersion getLatestReleaseVersion(String groupId, String artifactId) {
+    VersionRange range = null;
+    return getLatestReleaseVersion(idToKey(groupId, artifactId), range);
   }
 
   /**
    * Returns the latest published release artifact version in the version range,
    * or null if there is no such version.
    */
-  public ArtifactVersion getLatestArtifactVersion(
+  public ArtifactVersion getLatestReleaseVersion(
```

Comment:
Now that the name becomes identical to the private method below, this feels like an extra indirection that is unnecessary. We can get rid of this method if we push `idToKey()` down to the private method and make it public.

Reply:
done

Reply:
Commit not pushed.